Formula/pcp.rb

```diff
@@ -0,0 +1,20 @@
+class Pcp < Formula
+  desc "📦 Command-line peer-to-peer data transfer tool based on libp2p"
+  homepage "https://github.com/dennis-tra/pcp"
+  url "https://github.com/dennis-tra/pcp/archive/refs/tags/v0.4.0.tar.gz"
+  sha256 "09bf477afcca5aabd617c90f012063786715ab5715cce77a72f2a0ae758585ea"
+  license "Apache-2.0"
+  head "https://github.com/dennis-tra/pcp.git"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-X main.RawVersion=0.4.0 -X main.ShortCommit=7f638fe"
```

Comment:
```suggestion
    ldflags = "-X main.RawVersion=#{version} -X main.ShortCommit=#{Utils.git_short_head(length: 7)}"
```
You'll need to switch the tarball to a `git` checkout to use `Utils.git_short_head` though.

Reply:
Thanks for the suggestion! I updated the PR